app/experimenter/experiments/models.py

```diff
@@ -126,6 +126,11 @@ class Experiment(ExperimentConstants, models.Model):
         default=ExperimentConstants.ANALYSIS_DEFAULT, blank=True, null=True
     )
     analysis_owner = models.CharField(max_length=255, blank=True, null=True)
+
+    survey = models.NullBooleanField(default=None, blank=True, null=True)
+    survey_urls = models.TextField(blank=True, null=True)
+    survey_instructions = models.TextField(blank=True, null=True)
```

Comment:
I admit that I don't know 100% how Jared prefers these things, but I would probably have tried to avoid `null` and made them default to empty string. For `survey` I guess we have to leave null rather than default to "no".

Reply:
Good point, @jaredkerim can weigh in too.

Reply:
So there's a lot of cases where we set fields to blank/null=True and let the default be null/None so that we can distinguish between "the user has not yet interacted with this field" and "the user did interact and entered some falsy value".  So for risk questions, we don't want to default to "No" becuase we can't know that's true and we want to force the user to make an explicit declaration.

In this case though, I think it is sensible to have the survey required field default to No, since defaulting to that doesn't introduce risk to the experiment.  Because I think our behaviour is that if an experiment owner doesn't interact with the survey fields at all the experiment should proceed as normal.

Edit: confirmed with @shell1

Reply:
I made this change... hopefully you can check to make sure I understood you correctly.